exporter/awss3exporter/config_test.go

```diff
@@ -124,32 +135,42 @@ func TestConfig_Validate(t *testing.T) {
 			config: func() *Config {
 				c := createDefaultConfig().(*Config)
 				c.S3Uploader.Region = ""
+				c.S3Uploader.S3Bucket = ""
+				c.S3Uploader.Endpoint = ""
 				return c
 			}(),
-			errExpected: multierr.Append(errors.New("region is required"),
-				errors.New("bucket is required")),
+			errExpected: errors.New("endpoint and region should be provided, or bucket amd region"),
```

Comment:
this change is unexpected

Reply:
Good point, will revert.

Reply:
[5a23120](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32815/commits/5a23120ccee51aaa96f025c667d2d2bb7c8498fd)